apps/account-api/src/controllers/v1/handles-v1.controller.ts

```diff
@@ -1,20 +1,33 @@
-import { Controller, Get, Post, HttpCode, HttpStatus, Logger, Param, HttpException, Body } from '@nestjs/common';
+import {
+  Controller,
+  Get,
+  Post,
+  HttpCode,
+  HttpStatus,
+  Logger,
+  Param,
+  HttpException,
+  Body,
+  UseGuards,
+} from '@nestjs/common';
 import { ApiBody, ApiOkResponse, ApiOperation, ApiTags } from '@nestjs/swagger';
-import { TransactionType } from '#lib/types/enums';
-import { HandlesService } from '#api/services/handles.service';
-import { EnqueueService } from '#lib/services/enqueue-request.service';
+import { TransactionType } from '#account-lib/types/enums';
+import { HandlesService } from '#account-api/services/handles.service';
+import { EnqueueService } from '#account-lib/services/enqueue-request.service';
 import {
   ChangeHandlePayloadRequest,
   ChangeHandleRequest,
   CreateHandleRequest,
   HandleRequestDto,
-} from '#lib/types/dtos/handles.request.dto';
-import { TransactionResponse } from '#lib/types/dtos/transaction.response.dto';
+} from '#account-lib/types/dtos/handles.request.dto';
+import { TransactionResponse } from '#account-lib/types/dtos/transaction.response.dto';
+import { HandleResponseDto } from '#account-lib/types/dtos/accounts.response.dto';
+import { ReadOnlyGuard } from '#account-api/guards/read-only.guard';
 import { u8aToHex, u8aWrapBytes } from '@polkadot/util';
-import { HandleResponseDto } from '#lib/types/dtos/accounts.response.dto';
 
 @Controller('v1/handles')
 @ApiTags('v1/handles')
+@UseGuards(ReadOnlyGuard)
```

Comment:
Wasn't expecting this change

Reply:
When I did a merge from latest `main` to resolve branch conflicts, there were a couple of endpoints added that are 'writeable' endpoints that need to have that guard. Was missed in the PR that added them.

Reply:
Is this a way to disable some endpoints without removing them from code? Since I can see some POST endpoint under this controller?

Reply:
Yes; this disables PUT, POST, and DELETE endpoints with a `403 FORBIDDEN` response.

Reply:
part of [this PR](https://github.com/ProjectLibertyLabs/gateway/pull/446)